src/main/java/ca/corefacility/bioinformatics/irida/ria/web/sequencingRuns/SequencingRunAjaxController.java

```diff
@@ -47,11 +47,12 @@ public TableResponse listSequencingRuns(@RequestBody SequencingRunsListRequest s
 		Page<SequencingRun> list = sequencingRunService.list(sequencingRunsListRequest.getCurrent(),
 				sequencingRunsListRequest.getPageSize(), sequencingRunsListRequest.getSort());
 
-		List<TableModel> runs = list.getContent()
-				.stream()
-				.map(s -> new SequencingRunModel(s, messageSource.getMessage("sequencingruns.status." + s.getUploadStatus()
-						.toString(), new Object[] {}, locale)))
-				.collect(Collectors.toList());
+		List<TableModel> runs = new ArrayList<>();
+		for (SequencingRun run : list.getContent()) {
+			runs.add(new SequencingRunModel(run, messageSource.getMessage(
+					"sequencingruns.status." + run.getUploadStatus()
+							.toString(), new Object[] {}, locale)));
+		}
 
 		return new TableResponse(runs, list.getTotalElements());
```

Comment:
This should be parameterized `<...>`

Reply:
Along with the method signature.

Reply:
Fixed in f1b356de18f58d0f98e1bb9bc8873c9f3be62be3